sensors/src/jira_issue_sensor.py

```diff
@@ -0,0 +1,69 @@
+from st2reactor.sensor.base import PollingSensor
+from jira_api.jira_issue import search_issues
+from structs.jira.jira_account import JiraAccount
+import jira
+
+
+class JiraIssueSensor(PollingSensor):
+    """
+    class to implement an active Sensor to query a JIRA project
+    for very specific type of JIRA tickets (issues) and perform
```

Comment:
More specific than specific?
```suggestion
    for specific type of JIRA tickets (issues) and perform
```